src/mongod_helpers.py

```diff
@@ -53,18 +53,54 @@ def check_server_info(self, client: MongoClient):
         """
         return client.server_info()
 
-    def is_ready(self, standalone: bool = False) -> bool:
-        """Is the MongoDB server ready to services requests.
+    def check_replica_status(self) -> str:
+        """Retrieves the status of the current replica."""
+        # connect to mongod and retrieve replica set status
+        client = self.client(standalone=True)
+        try:
+            status = client.admin.command("replSetGetStatus")
+        except (ConnectionFailure, ConfigurationError, OperationFailure) as e:
+            logger.error("Failed to check the replica status, error: %s", e)
+            raise e
```

Comment:
are we confident that these are the only exceptions that can occur when executing this command? also, should we be `raise`ing instead of `raise e` to preserve the original stack trace?